tests/next_tests/integration_tests/feature_tests/ffront_tests/test_import_from_mod.py

```diff
@@ -63,12 +63,12 @@ def field_op(f: cases.IField):
             f_i_k = gtx.broadcast(f, (cases.IDim, cases.KDim))
             return f_i_k
 
-    with pytest.raises(ValueError):
```

Comment:
I would also look in a next PR, which of these cases could be supported easily now.